docs/explore/what-is-redis/index-what-is-redis.mdx

```diff
@@ -0,0 +1,155 @@
+hide_title: true
+sidebar_label: What is Redis?
+slug: /explore/what-is-redis
+authors: [will]
+keywords:
+  - Redis
+  - in-memory database
+  - caching
+  - session store
+  - leaderboard
+  - pub/sub
+  - real-time apps
+  - gaming
+---
+
+# What is Redis?
+
+Redis is an open-source, in-memory, key-value data store most commonly used as a database, cache, message broker, and queue. Redis delivers sub-millisecond response times, enabling fast and powerful real-time applications in industries such as gaming, fintech, ad-tech, social media, healthcare, and IoT. Redis is the [most-loved database](https://insights.stackoverflow.com/survey/2021) by developers for five years running. Developers love Redis because of its ease of use, performance, and scalability. There is a Redis client available for use in every popular modern programming language. This coupled with the performance benefits makes Redis the most popular choice for caching, session management, gaming, fraud detection, leaderboards, real-time analytics, geospacial indexing, ride-sharing, social media, and streaming applications.
+
+### Redis Anywhere
+
+Redis Enterprise is the only true datastore built for hybrid and multi-cloud applications. [Get started with Redis Enterprise Cloud](https://developer.redis.com/create/rediscloud).
+
+### Redis on AWS
+
+You might be familiar with Amazon ElastiCache for Redis. It is a Redis-compatible cache service that is available on AWS. Redis Enterprise Cloud on AWS is a fully-managed Redis Enterprise as a service and supports Redis as a cache and a database. Learn more about [Redis on AWS](https://developer.redis.com/create/aws/redis-on-aws).
+
+### Redis on Other Cloud Providers
+
+- [Redis on Google Cloud](https://developer.redis.com/create/gcp)
+- [Redis on Azure](https://developer.redis.com/create/azure)
+- [Redis on Heroku](https://developer.redis.com/create/heroku)
+
+## Benefits of Redis
+
+### Performance
+
+The primary benefit of Redis is its sub-millisecond queries. Redis runs in-memory, which enables low-latency and high throughput. Running in-memory means requests for data do not require a trip to disk. This leads to an order of magnitude more operations and faster response times. Redis is one of the only databases that supports millions of operations per second.
+
+### Flexible Data Structures
+
+Redis is a multi-model database, and provides several built-in data structures such as:
+
+- Strings - any text or binary data (512MB max)
+- Hashes - field-value pairs that most commonly represent objects
+- Lists - a collection of Strings ordered by when they were added as a linked list. Useful for queues and "latest updates" for social media posts
+- Sets - an unordered collection of Strings with the ability to intersect, union, and diff against other Sets
+- Sorted Sets - a mix between a Set and a Hash, where the Set is ordered by a value (score). Useful for leaderboards, games, and analytics.
```

Comment:
@wjohnsto @ajeetraina this is a strange way to describe a sorted set.  I'd drop the part about a Hash.  Probably best to say something like "Sorted sets - similar to a Redis Set, the Sorted Set is a collection of unique string members.  In a Sorted Set, each member is associated with a score that can be used to sort the collection".